test/assembly/MigrationTest.java

```diff
@@ -95,14 +95,18 @@ public void testMigration() throws InterruptedException, TimeoutException, IOExc
 
     @Test
     public void testSchemaExport() throws InterruptedException, IOException, TimeoutException {
+        System.out.println("Loading schema... ");
         loadSimulationSchema("schematest");
```

Comment:
and we wanted to load the initial schema here to make sure there's no error in it right?

Reply:
Done.